src/documentation/markdown/2017/panel-data-fields.md

```diff
@@ -0,0 +1,126 @@
+## Public Panel data fields with values and definitions
+
+### [activity\_year](#activity_year)
+**Description:** The collection year of the HMDA data to which the Panel file relates
+
+**Values:**
+
+- 2017
+
+### [lei](#lei)
+**Description:** A financial institution’s Legal Entity Identifier
+
+**Values:**
+
+- Varying values
+
+### [tax\_id](#tax_id)
+**Description:** The federal tax ID of the institution
+
+**Values:**
+
+- Varying values
+
+### [agency\_code](#agency_code)
+**Description:** The integer code corresponding to an institution's regulatory agency
+
+**Values:**
+  - 1: Office of the Comptroller of the Currency (OCC)
+  - 2: Federal Reserve System (FRB)
+  - 3: Federal Deposit Insurance Corporation (FDIC)
+  - 4: Office of Thrift Supervision (OTS)
+  - 5: National Credit Union Administration (NCUA)
+  - 7: Department of Housing and Urban Development (HUD)
+  - 9: Consumer Financial Protection Bureau (CFPB)
+
+### [id\_2017](#id_2017)
+**Description:** The 2017 HMDA Platform primary identifier for an institution. This identifier is used in naming modified LAR files on [this page](https://ffiec.cfpb.gov/data-publication/modified-lar/2017)
+
+**Values:**
+
+- Varying values
+
+### [arid\_2017](#arid_2017)
+**Description:** The concatenation of an institution's 2017 Agency Code and Respondent ID. In order to match between 2017 and 2018, take the arid_2017 column from the 2018 panel and use this to join to the concatenation of the agency code and respondent ID in the 2017 panel. Respondent ID in 2017 and prior is not a unique value and must be joined to agency code to generate a primary key for the HMDA dataset. For 2018 and forward, LEI will be the primary key.
```

Comment:
please add the following:

for more information on institution identifiers, please see: https://ffiec.cfpb.gov/documentation/2020/identifiers-faq/